src/components/value-list/utils.ts

```diff
@@ -0,0 +1,56 @@
+import { getItemIndex } from "../pick-list/shared-list-logic";
+import { DragStatus } from "./interfaces";
+import { ValueList } from "./value-list";
+
+export function getScreenReaderText(
+  item: HTMLCalciteValueListItemElement,
+  status: DragStatus,
+  valueList: ValueList<HTMLCalciteValueListItemElement>
+): string {
+  const {
+    items,
+    intlDragHandleStart,
+    intlDragHandleActivated,
+    intlDragHandleNewPosition,
+    intlDragHandleCurrentPosition
+  } = valueList;
+  const total = items.length;
+  const position = getItemIndex(valueList, item) + 1;
+  switch (status) {
+    case "start":
+      return intlDragHandleStart
+        ? intlDragHandleStart
+        : `${item.label} ,press space and use arrow keys to re-order content. current position ${position} of ${total}`;
+    case "activated":
+      return intlDragHandleActivated
+        ? intlDragHandleActivated
+        : `Reordering ${item.label} ,current position ${position} of ${total}`;
+    case "newPosition":
+      return intlDragHandleNewPosition
+        ? intlDragHandleNewPosition
+        : `${item.label} , new position ${position} of ${total}. press space to confirm`;
+    case "currentPosition":
+      return intlDragHandleCurrentPosition
+        ? intlDragHandleCurrentPosition
+        : `${item.label} ,current position ${position} of ${total}`;
```

Comment:
There are punctuation/capitalization issues in the default strings that need to be fixed:

1. `${item.label}, press space and use arrow keys to reorder content. Current position ${position} of ${total}`
2. `${item.label}, current position ${position} of ${total}`
3. `${item.label}, new position ${position} of ${total}. Press space to confirm`
4. `Reordering ${item.label}, current position ${position} of ${total}`

@geospatialem Can you give this an editorial pass as well?

Reply:
@jcfranco's recommendations are sound to me. Perhaps periods at the end of each statement, but the contents and edits are great.